src/Bicep.Cli.IntegrationTests/ValidateParamsCommandTests.cs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Bicep.Core.Configuration;
+using Bicep.Core.Samples;
+using Bicep.Core.UnitTests;
+using Bicep.Core.UnitTests.Assertions;
+using Bicep.Core.UnitTests.Features;
+using Bicep.Core.UnitTests.Utils;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Bicep.Cli.IntegrationTests
+{
+
+    [TestClass]
+    public class ValidateParamsCommandTests : TestBase
+    {
+        
+        [NotNull]
+        public TestContext? TestContext { get; set; }
+
+        [TestMethod]
+        [DoNotParallelize]
+        public async Task Validate_params_with_correct_parameter_values_runs_successfully()
+        {
+            var bicepPath = FileHelper.SaveResultFile(TestContext, "main.bicep", """
+param intParam int
+
+@allowed([
+    'foo'
+    'bar'
+])
+param strParam string
+
+param boolParam bool
+
+param arrParam array
+
+param objParam object
+""");
+
+            var paramsInput = """
+{
+  "intParam": 1,
+  "strParam": "foo",
+  "boolParam": true,
+  "arrParam": [1, 2, 3],
+  "objParam": {
+    "firstProp": "foobar",
+    "secondProp": [1, 2, 3] 
+  }
+}
+""";       
```

Comment:
```suggestion
            var bicepPath = FileHelper.SaveResultFile(TestContext, "main.bicep", """
                param intParam int

                @allowed([
                    'foo'
                    'bar'
                ])
                param strParam string

                param boolParam bool

                param arrParam array

                param objParam object
                """);

            var paramsInput = """
                {
                  "intParam": 1,
                  "strParam": "foo",
                  "boolParam": true,
                  "arrParam": [1, 2, 3],
                  "objParam": {
                    "firstProp": "foobar",
                    "secondProp": [1, 2, 3] 
                  }
                }
                """;       
```

The neat thing about `"""`-delimited strings is that the terminating `"""` defines what column will be the left margin for the string. In this snippet above, because the ending `"""` starts at column 16, 16 leading spaces will be trimmed from every line in the string.